### ISSUE TITLE
1.5 vs 1.4.* "mup deploy --cached-build" differences

### ISSUE THREAD

**Issue by Batistleman** [zodern/meteor-up#1154]
If I understand correctly, in mup 1.4.* the `mup deploy --cached-build` expected a `tar.gz` file, while 1.5 expects the `tar.gz` file extracted in a bundle directory already? Is that the only difference?

**Comment by zodern** [comment]
`--cached-build` should work the same in both versions. It expects a `tar.gz` file on the computer deploying from.

**Comment by Batistleman** [comment]
Strange, this is the error I get: 

```
Not building app. Using build from previous deploy at
<path>/bundles/
Error: ENOENT: no such file or directory, open '<path>/bundle/star.json'
    at Object.openSync (fs.js:450:3)
    at Object.readFileSync (fs.js:350:35)
    at getNodeVersion (<other path>/node_modules/mup/lib/plugins/meteor/utils.js:115:26)
```

**Comment by zodern** [comment]
You are correct. It does use the star.json file in the bundle to get the correct node version. I can look into an alternative implementation so you don't need both the build folder and the `.tar.gz` file.

**Comment by Batistleman** [comment]
For me this is fine, I'll script it to extract the `.tar.gz`.

So both the `.tar.gz` and extracted folder should be present?

PS. Thanks for the awesome work on 1.5! 🎉

**Comment by zodern** [comment]
This will be fixed in Mup 1.5.2 to only require the `.tar.gz` file.

**Comment by Batistleman** [comment]
Great, thanks!
